apps/netutils/wifi/slsi_wifi_api.c

```diff
@@ -1316,14 +1315,13 @@ void slsi_monitor_thread_handler(void *param)
 					break;
 				case SLSI_WIFIAPI_STATE_AP_ENABLED:
 					if (slsi_event_recieved(result, AP_STA_CONNECTED)) {
-						result += sizeof(AP_STA_CONNECTED) - 1;
-						if (strlen(result) >= 17) {	// bssid is a 17 character string
-							memcpy(&(reason.bssid), result, 17);	// Exclude null-termination
-						}
 						g_num_sta_connected++;
 						if (g_num_sta_connected == 1) {
 							g_state = SLSI_WIFIAPI_STATE_AP_CONNECTED;
-
+						}
+						result += sizeof(AP_STA_CONNECTED)-1;
```

Comment:
where spaces are needed ? could tell me in more detail ?

                        g_num_sta_connected++;
                        if (g_num_sta_connected == 1) {
                            g_state = SLSI_WIFIAPI_STATE_AP_CONNECTED;
                        }
                        result += sizeof(AP_STA_CONNECTED)-1;